lib/vintage_net_wizard/activity_monitor.ex

```diff
@@ -0,0 +1,52 @@
+defmodule VintageNetWizard.ActivityMonitor do
```

Comment:
How about calling this a Watchdog? It feels a little more appropriate since the module wants to get `pet` frequently or it stops the wizard.